src/index.ts

```diff
@@ -187,6 +197,20 @@ function getLatestSliderDate(cb: (date: Date) => void) {
   });
 }
 
+// TODO: Finish
+function getLatestMeteosatDate() {
+  json(`${METEOSAT_BASE_URL}msg`, (error, data: {url: string, date: string}) => {
+    /* Throws:
+     * Failed to load https://meteosat-url.appspot.com/msg: No
+     * 'Access-Control-Allow-Origin' header is present on the requested
+     * resource. Origin 'chrome-extension://nbcijgaiaigkfephnhbhkaepnfkhpgkl'
+     * is therefore not allowed access.
+     */
+    if (error) { throw error; }
```

Comment:
This is what's causing me grief right now. Looks like a CORS problem, but I don't see any difference between this call and the other ones on e.g. line 194. I sadly don't have too much experience with web apps; do you have an idea? Could it be that the web service needs to provide additional headers, or is it maybe an issue of my environment?

Reply:
I fixed it. Try again!

Reply:
Thanks! That puts me back on track. About to be flying again for a day but I'm still on it :)